File: palindrome.js
function palindrome(){
    var num = parseInt(document.getElementById("input").value)
    console.log(num)
    for (let i = 1; i <= num; i++){
         console.log(" ".repeat(num-i)+"*".repeat((2*i)-1));
    }
    for (let j = num-1; j >0; j--){
        console.log(" ".repeat(num-j)+"*".repeat((2*j)-1));
    }
}

File: palindrome.py
def palindrome():
    row = input("For rows: ")
    for i in range(0,row+1):
        print(" "*(row-i),"*"*((2*i)-1))
    for j in range(row-1,0,-1):
        print(" "*(row-j),"*"*((2*j)-1))